cmd/removeEnumerator.go

```diff
@@ -87,7 +87,27 @@ func newRemoveEnumerator(cca *CookedCopyCmdArgs) (enumerator *CopyEnumerator, er
 		jobsAdmin.JobsAdmin.LogToJobLog(message, common.LogInfo)
 	}
 
-	transferScheduler := newRemoveTransferProcessor(cca, NumOfFilesPerDispatchJobPart, fpo)
+	targetURL, _ := cca.Source.String()
+	from := cca.FromTo.From()
+	if !from.SupportsTrailingDot() {
+		cca.trailingDot = common.ETrailingDotOption.Disable()
+	}
+	options := createClientOptions(common.AzcopyCurrentJobLogger)
+	var fileClientOptions any
```

Comment:
why is this type any?

Reply:
Leaves room if blob/dfs get their specific options. For now only AzFiles has, and the var name reflects this.